app/assets/stylesheets/fuel/components/_editor.scss

```diff
@@ -0,0 +1,84 @@
+
+$editor-border: thin solid $grey-base;
+
+#toolbar {
+  @include clearfix;
+  background-color: $white-dark;
+  border: $editor-border;
+  border-bottom: none;
+  padding: 0;
```

Comment:
Properties should be ordered background-color, border, border-bottom, margin, padding, position